packages/mui-joy/src/styles/defaultTheme.ts

```diff
@@ -1,5 +1,3185 @@
-import extendTheme from './extendTheme';
+import {
+  createBreakpoints,
+  createSpacing,
+  unstable_styleFunctionSx as styleFunctionSx,
+  unstable_createGetCssVar as systemCreateGetCssVar,
+} from '@mui/system';
+import shouldSkipGeneratingVar from './shouldSkipGeneratingVar';
+import sxConfig from './sxConfig';
+import type { Components } from './components';
+import type { Theme, FontSize, SxProps, SupportedColorScheme } from './types';
 
-const defaultTheme = extendTheme();
+const defaultTheme = {
```

Comment:
Just to confirm, is this a copy of the actual generated object by calling the previous `extendTheme()` function?